packages/components/app/styles/components/alert.scss

```diff
@@ -164,14 +173,10 @@
 }
 
 .hds-alert--type-compact {
-  .hds-alert__icon {
```

Comment:
mmm, seems the linter is changing code here. we should avoid this

Reply:
this was because this selector is technically being duplicated. could definitely disable that rule if we don't want it.

https://github.com/hashicorp/design-system/pull/356/commits/81d6fff02b9fa2d060a39c5093618f36d701ea02